src/analysis/normed_space/inner_product.lean

```diff
@@ -1425,6 +1518,13 @@ instance : finite_dimensional 𝕜 (euclidean_space 𝕜 ι) := by apply_instanc
 lemma findim_euclidean_space_fin {n : ℕ} :
   finite_dimensional.findim 𝕜 (euclidean_space 𝕜 (fin n)) = n := by simp
 
+/-- A basis on `ι` for a finite-dimensional inner product space induces a continuous linear
+equivalence with `euclidean_space 𝕜 ι`.  If the basis is orthonormal, this continuous linear
+equivalence is an isometry, but we don't prove that here. -/
```

Comment:
The comment is a little bit misleading as the space does not need to be an inner product space for the definition to make sense. Maybe

```suggestion
/-- A basis on `ι` for a finite-dimensional space induces a continuous linear equivalence 
with `euclidean_space 𝕜 ι`.  If the basis is orthonormal in an inner product space, this continuous 
linear equivalence is an isometry, but we don't prove that here. -/
```